### PR TITLE
Python metadata changes for NI-DCPower 20.6 release

### PR DESCRIPTION
**Pull request by nisundar**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [x] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable

- [ ] ~~I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?
* API parity with NI-DCPower 20.6.0 by adding Merged Channels and Shutdown Triggers support
* Update documentation with the latest 'Driver Version Tested Against'

### List issues fixed by this Pull Request below, if any.
n/a

### What testing has been done?
System tests will be run by nimi-bot

### CODE REVIEW

**Comment by marcoskirsch** [comment]
In the description, you can delete "TODO" once you've done it.

File: src/nidcpower/metadata/config.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-DCPower API metadata version 20.6.0d7
+# This file is generated from NI-DCPower API metadata version 20.7.0d7
 config = {
-    'api_version': '20.6.0d7',
+    'api_version': '20.7.0d7',
```
Comment (marcoskirsch) [inline src/nidcpower/metadata/config.py:4]: Can't you use 20.6.0f0?

**Review by marcoskirsch — CHANGES_REQUESTED** [review]
There is metadata for things like "Driver version tested against". When do you plan to tackle that?

File: src/nidcpower/metadata/config.py (hunk 1 above)
Comment (nisundar) [reply src/nidcpower/metadata/config.py:4]: Yes. This is a mistake. I'll fix it.

**Comment by sbethur** [comment]
test this please

**Comment by sbethur** [comment]
ok to test

File: src/nidcpower/metadata/config.py (hunk 1 above)
Comment (nisundar) [reply src/nidcpower/metadata/config.py:4]: To fix this the right way, we have cherry pick some metadata fixes we did and build a 20.6.0f1.

Comment (marcoskirsch) [reply src/nidcpower/metadata/config.py:4]: Not worth it. Use 20.7.0d7 or whatever it is you need.

**Comment by sbethur** [comment]
> There is metadata for things like "Driver version tested against". When do you plan to tackle that?

Change needs to be done in https://github.com/ni/nimi-python/blob/be1d5aa2eea7d22e223d64ae7012bf8e3ef79eea/src/nidcpower/metadata/config_addon.py#L4 and code generator should be run to update the README.

File: src/nidcpower/metadata/attributes.py [hunk 2]
```diff
@@ -102,7 +102,7 @@
     },
     1050511: {
         'access': 'read only',
-        'channel_based': False,
+        'channel_based': True,
```
Comment (sbethur) [inline src/nidcpower/metadata/attributes.py:105]: It's possible these changes to `channel_based` key were made in internal metadata for the CXP feature. If that's the case, these changes should happen when the CXP support is added in nimi-python.

Comment (nisundar) [reply src/nidcpower/metadata/attributes.py]: I noticed that we some existing tech-debt in this area (TechinicalDebt# 1131303). I have manually reverted these changes.

**Comment by sbethur** [comment]
@nisundar, please include the updated https://github.com/ni/nimi-python/blob/master/CHANGELOG.md in the PR

File: README.rst [hunk 3]
```diff
@@ -70,7 +70,7 @@ NI-DCPower Python API Status
 +-------------------------------+--------------------------+
 | NI-DCPower (nidcpower)        |                          |
 +===============================+==========================+
-| Driver Version Tested Against | 20.5.0                   |
+| Driver Version Tested Against | 20.6.0                   |
```
Comment (sbethur) [inline README.rst:73]: Mention about this change in the PR description

**Review by sbethur — APPROVED** [review]

**Review by marcoskirsch — APPROVED** [review]

**Pull request merged**